cluster.go

```diff
@@ -47,32 +47,34 @@ import (
 
 type mongoCluster struct {
 	sync.RWMutex
-	serverSynced  sync.Cond
-	userSeeds     []string
-	dynaSeeds     []string
-	servers       mongoServers
-	masters       mongoServers
-	references    int
-	syncing       bool
-	direct        bool
-	failFast      bool
-	syncCount     uint
-	setName       string
-	cachedIndex   map[string]bool
-	sync          chan bool
-	dial          dialer
-	maxSocketUses int
+	serverSynced       sync.Cond
+	userSeeds          []string
+	dynaSeeds          []string
+	servers            mongoServers
+	masters            mongoServers
+	references         int
+	syncing            bool
+	direct             bool
+	failFast           bool
+	syncCount          uint
+	setName            string
+	cachedIndex        map[string]bool
+	sync               chan bool
+	dial               dialer
+	maxSocketUses      int
```

Comment:
do you still plan to use maxSocketUses?  If not, maybe it makes sense to remove it?  This is something I added previously and it hasn't been upstreamed.